core/services/job/job_orm_test.go

```diff
@@ -150,19 +150,26 @@ func TestORM(t *testing.T) {
 		require.NoError(t, orm.RecordError(jobSpec.ID, ocrSpecError1))
 		require.NoError(t, orm.RecordError(jobSpec.ID, ocrSpecError2))
 
-		expected1ID := specErrors[1].ID + 1
-		expected2ID := specErrors[1].ID + 2
-		specErr1, err := orm.FindSpecError(expected1ID)
+		var updatedSpecError []job.SpecError
+
+		err = db.Select(&updatedSpecError, "SELECT * FROM job_spec_errors ORDER BY id ASC")
+		require.NoError(t, err)
```

Comment:
This should be an assert, and stop further execution of the test.
Since if we are unable to fetch errors from the db, there is no point of proceeding further.
More seriously, in below lines 159-162, you are directly referencing updatedSpecError by index 2 and 3. If the query had failed, those direct references will crash the test suite entirely.